src/commands/uninstall.rs

```diff
@@ -16,6 +17,64 @@ pub async fn uninstall(
     module_names: Vec<String>,
     flags: &StandardOptions,
 ) -> Result<(), SysexitsError> {
+    if let Ok(mut module_bin_dir) = tokio::fs::read_dir(asimov_root().join("libexec")).await {
+        while let Ok(Some(entry)) = module_bin_dir.next_entry().await {
+            let path = entry.path();
+            let Some(name) = path.file_name().map(|n| n.to_string_lossy()) else {
+                continue;
+            };
+            let mut parts = name.split('-');
+            if !parts.next().is_some_and(|prefix| prefix == "asimov") {
+                continue;
+            }
+            if !parts
+                .next()
+                .is_some_and(|name| module_names.iter().any(|module| name == module))
```

Comment:
That's clever, but perhaps too clever: module manifests do include the program (flow) binary names, so a much simpler implementation would be to load the module manifest and remove those specific programs and then the manifest. (And in case the manifest can't be found, then the module isn't installed.)